package.json

```diff
@@ -33,7 +33,8 @@
     ".": "./lib/index.js",
     "./els": "./lib/els.js",
     "./effects": "./lib/effects.js",
-    "./steps": "./lib/steps.js"
+    "./steps": "./lib/steps.js",
+    "./step": "./lib/step.js"
```

Comment:
how about adding step export to `index.js`?

```
const { step } = require('codeceptjs')
```

this is how we access all internal API 

`codeceptjs/steps` is userland API which is used inside tests, not custom helpers, or plugins